docs/_src/language/basic.md

```diff
@@ -259,9 +340,33 @@ explore airports
   )
 | project top 10 order by 4 desc
   by_county.county
-  airport_count
-  county_airport_count is by_county.airport_count
-  percent_airports_in_county is by_county.airport_count/airport_count * 100
+  airports_in_county is by_county.airport_count
+  airports_in_state is airport_count
+  portion_in_county is by_county.airport_count / airport_count
```

Comment:
I didn't want to do _percent_ because I hate when percents are formatted in output tables without the % sign. And to do that, we'd either need a data style for it, which we don't really talk about in this document, or to do some gross string formatting. 

Maybe this is just a personal peeve, but I feel like this is clear enough with proportion rather than percent.

Reply:
I fall on the other side and think most people think and talk in percents vs proportion. Definitely nicer with percent than without, but I just don't think people think in .17 of county, they think 17% of the county

Reply:
I suppose not that important either way, but if you go with proportion, should change "portion" to "proportion" in field name I think?

Reply:
Just added a quick `percent` renderer. I think we can hand wave the renderer here. "It's magically rendered as a percent"